frontend/src/components/Forms/InterestsForm/index.js

```diff
@@ -7,25 +7,7 @@ import Checkbox from "components/Checkbox";
 
 import styles from "./index.module.css";
 
-const InterestsForm = ({
-  selectedInterests,
-  setSelectedInterests,
-  addedInterest,
-  setAddedInterest,
-  error,
-}) => {
-  const handleInterest = ({ target }) => {
-    if (selectedInterests.includes(target.id)) {
-      setSelectedInterests((prevArray) =>
-        prevArray.filter((item) => item !== target.id)
-      );
-    } else {
-      setSelectedInterests(
-        (prevArray) => (prevArray = [...prevArray, target.id])
-      );
-    }
-  };
-
+const InterestsForm = ({ handleChange, addedInterest, error }) => {
```

Comment:
Just nitpicking, but `extraInterest`? You don't have to change it. I just want you to know that I had to think twice of this one was.

Reply:
Oh okay, yeah that is more explicit indeed, will make sure to change that as well!